core/Updater/Migration/Custom.php

```diff
@@ -20,16 +20,18 @@ class Custom extends Migration
 {
     private $callback;
     private $toString;
+    private $args;
 
-    public function __construct($callback, $toString)
+    public function __construct($callback, $toString, $args = null)
```

Comment:
We should directly type hint this if we expect an array (when using `call_user_func_array`)
```suggestion
    public function __construct($callback, $toString, array $args = [])
```